packages/stargate/src/stargateclient.ts

```diff
@@ -85,20 +85,19 @@ function uint64FromProto(input: number | Long | null | undefined): Uint64 {
   return Uint64.fromString(input.toString());
 }
 
-function accountFromProto(input: cosmos.auth.IBaseAccount, prefix: string): Account {
+function accountFromProto(input: cosmos.auth.v1beta1.IBaseAccount): Account {
   const { address, pubKey, accountNumber, sequence } = input;
-  // Pubkey is still Amino-encoded in BaseAccount (https://github.com/cosmos/cosmos-sdk/issues/6886)
-  const pubkey = pubKey && pubKey.length ? decodeAminoPubkey(pubKey) : null;
+  const pubkey = decodePubkey(pubKey);
```

Comment:
This diff closes https://github.com/CosmWasm/cosmjs/issues/363, right?

Reply:
Yes :)